python/glutil.cpp

```diff
@@ -142,12 +142,19 @@ void register_glutil(py::module &m) {
              D(GLShader, drawIndexed), py::arg("type"),
              py::arg("offset"), py::arg("count"))
         .def("setUniform", &setUniformPy, py::arg("name"),
-             py::arg("value"), py::arg("warn") = true);
+             py::arg("value"), py::arg("warn") = true)
+        .def("glEnable", [](GLShader, py::int_ cap) {
```

Comment:
Actually, it would be nicer to turn these into ``gl.def(..)`` at the bottom. These are really static GL calls that aren't related to GLShader, so they should be in the ``gl`` namespace.